Extend `People/Mapping Assistance Team` with references and missing details (#11705)
ppy/osu-wiki: wiki/People/Mapping_Assistance_Team/en.md

# Mapping Assistance Team

> The **Mapping Assistance Team** (***MAT***) was a group responsible for assessing the quality of [pending](/wiki/Beatmap/Category#wip-and-pending) [beatmaps](/wiki/Beatmap) and recommending them for [ranking](/wiki/Beatmap/Category#ranked) in the [forum modding](/wiki/Modding/Forum_modding) system. Previously, mappers would go directly to a member of the [Beatmap Appreciation Team](/wiki/People/Beatmap_Appreciation_Team) (*BAT*) for beatmap moderation, but now mappers would approach a MAT instead. MATs would serve as a buffer for quality assurance, before then passing on maps to the BAT. Their recommendations for beatmaps were passed to the BAT by placing a [proto-bubble](/wiki/Modding/Proto-bubble), and the BAT would then make the final decision on whether the beatmap should enter Ranked.

Reviewer: ```suggestion
The **Mapping Assistance Team** (***MAT***) was a group responsible for assessing the quality of [pending](/wiki/Beatmap/Category#wip-and-pending) [beatmaps](/wiki/Beatmap) and recommending them for [ranking](/wiki/Beatmap/Category#ranked) in the [forum modding](/wiki/Modding/Forum_modding) system. The MAT served as a first point of contact for mappers looking to rank their maps, where previously this would have been part of the role of the [Beatmap Appreciation Team](/wiki/People/Beatmap_Appreciation_Team) (*BAT*). MATs passed their recommendations for ranking to the BAT by placing a [proto-bubble](/wiki/Modding/Proto-bubble) on the beatmap, and the BAT would then make the final decision on whether the beatmap should enter Ranked.
```
i want to avoid saying "now" to describe the updated situation, i get what you mean but it should still be clear that everything here is in the past. also reworded a bit to not duplicate as much from the first sentence in the paragraph

> The MAT were overseen by a [MAT Leader](/wiki/People/Mapping_Assistance_Team/MAT_Leaders) who provided guidance and assistance where necessary. 

Reviewer 2: ```suggestion
The MAT were overseen by a [MAT Leader](/wiki/People/Mapping_Assistance_Team/MAT_Leaders) who provided guidance and assistance where necessary.
```

## History

> The MAT was formed on the 27th of December, 2009, with initial members being hand-picked by members of the BAT.[^1] The first interim leader of the MATs was ::{ flag=AU }:: [Ephemeral](https://osu.ppy.sh/users/102335).

Reviewer: ```suggestion
The MAT was formed on the 27th of December, 2009, with initial members being hand-picked by members of the BAT.[^1] The first interim leader of the MAT was ::{ flag=AU }:: [Ephemeral](https://osu.ppy.sh/users/102335).
```

Shortly afterwards, open applications were held, allowing all members of the general mapping and modding community to apply for a position in the MAT. On the 26th of February, 2010, the first new batch of MAT members were added from these applicants,[^2] based on a collective evaluation from members of the BAT and existing MAT. Further application cycles were held regularly to scout for new members, as well as provide feedback to unsuccessful applicants.

> On the 13th of April, 2010, the proto-bubble was officially discontinued for use by the MAT.[^3] After this change, the MAT role would not have any direct crontol over beatmaps, and would serve more as a title given to exceptional modders.[^4]

Reviewer 3: ```suggestion
On the 13th of April, 2010, the proto-bubble was officially discontinued for use by the MAT.[^3] After this change, the MAT role would not have any direct control over beatmaps, and would serve more as a title given to exceptional modders.[^4]
```

Following a gradual convergence of experience and responsibility between the two teams, on the 23rd of April, 2013, the MAT was dissolved,[^6] with all current members being promoted to the BAT and receiving the associated moderation tools and responsibilities.

## References

[^1]:  [Forum post announcing the creation of the MAT.](https://osu.ppy.sh/community/forums/topics/21856?n=1)
[^2]:  [Forum post announcing the addition of the new members.](https://osu.ppy.sh/community/forums/topics/25314?n=1)
[^3]:  [Forum post announcing the discontinuation of proto-bubbles for MATs.](https://osu.ppy.sh/community/forums/topics/28143?n=57)
[^4]:  [Forum post discussing the state of the MAT.](https://osu.ppy.sh/community/forums/topics/36277?n=1)
[^5]:  [Forum post announcing that MATs are able to bubble maps.](https://osu.ppy.sh/community/forums/topics/38405?n=1)

> [^6]:  [Forum post announcing the end of the MAT.](https://osu.ppy.sh/community/forums/topics/129165?n=1)

Reviewer 3: we have an established format for citations: https://osu.ppy.sh/wiki/en/Article_styling_criteria/Formatting#citations

> [^5]:  [Forum post announcing that MATs are able to bubble maps.](https://osu.ppy.sh/community/forums/topics/38405?n=1)

Reviewer: a few things here:
- as CI says already, you have some duplicate definition IDs here, looks like a mistake
- it would probably be better to give these definitions names instead of just numbering them
- when linking to a specific post, use the link of form `https://osu.ppy.sh/community/forums/posts/{post}`
- no need to say "(Restricted)" (also one of them is public anyway)

Author: I fixed the duplicate IDs and removed "(Restricted)", but trying to change the format of the link broke them and they sent me to completely unrelated posts. Also, I think the definitions being numbered are fine, they're just there to cite the source, nothing fancy is need.
